source/d_profileNP/bases/d_yes_no_window.cpp

```diff
@@ -0,0 +1,595 @@
+#include <d_profileNP/bases/d_yes_no_window.hpp>
+#include <dol/framework/f_profile.hpp>
+#include <dol/framework/f_profile_name.hpp>
+#include <dol/bases/d_audio.hpp>
+#include <dol/bases/d_game_com.hpp>
+#include <dol/bases/d_cursor_select.hpp>
+#include <dol/snd/snd_audio_mgr.hpp>
+#include <dol/bases/d_info.hpp>
+#include <dol/bases/d_s_world_map_static.hpp>
+#include <constants/message_list.h>
+#include <constants/game_constants.h>
+#include <constants/sound_list.h>
+
```

Comment:
```suggestion
/// @file

```
Docs added to this file will not be detected without this directive.